# Fix build failure due to automake 1.11.
# defsgen.py was installed twice
# Gentoo: https://bugs.gentoo.org/show_bug.cgi?id=279813
--- a/codegen/Makefile.am
+++ b/codegen/Makefile.am
@@ -18,7 +18,6 @@
 	docextract_to_xml.py \
 	docgen.py \
 	h2def.py \
-	defsgen.py \
 	createdefs.py \
 	mergedefs.py \
 	missingdefs.py \
